SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/ApplicationLifecycleHelper.cs

```diff
@@ -13,13 +13,13 @@ namespace Microsoft.AppCenter.Utils
     public class ApplicationLifecycleHelper : IApplicationLifecycleHelper
     {
         // Singleton instance of ApplicationLifecycleHelper
-        private static ApplicationLifecycleHelper _instance;
-        public static ApplicationLifecycleHelper Instance
+        private static IApplicationLifecycleHelper _instance;
+        public static IApplicationLifecycleHelper Instance
         {
             get { return _instance ?? (_instance = new ApplicationLifecycleHelper()); }
 
             // Setter for testing
```

Comment:
Here the comment was done right @jaelim-ms

Reply:
Let's keep internal and use InternalVisibleTo

Reply:
I am not blocking on that, but nice to have.

Reply:
@guperrot can you verify I did this correctly?

Reply:
looks good @michaelsmitha